src/inst/opts/consul.asc

```diff
@@ -0,0 +1,52 @@
+=== Custom Consul services
+
+Consul services check is based on defintion as per file `puppet-hpc/hieradata/common.yaml` extract:
```

Comment:
1/ There is a typo on word definition
2/ Do we really want to maintain a duplicate extract of puppet-hpc here? Just one fake example on custom additional service definition is fine.

Reply:
fixed